app/code/community/Reverb/ReverbSync/Helper/Sync/Category.php

```diff
@@ -69,22 +69,14 @@ public function addCategoriesToListingFieldsArray(array $fieldsArray, Mage_Catal
             return $fieldsArray;
         }
 
-        $category_slug_array = array();
-        $product_type_slug = null;
-
-        foreach($product_reverb_category_objects_array as $reverbCategory)
-        {
-            $category_slug_array[] = $reverbCategory->getReverbCategorySlug();
-            if (empty($product_type_slug))
-            {
-                $product_type_slug = $reverbCategory->getReverbProductTypeSlug();
-            }
-        }
+        $deepestReverbCategory = $this->getDeepestReverbCategory($product_reverb_category_objects_array);
+        $product_type_slug = $deepestReverbCategory->getData('reverb_product_type_slug');
+        $category_slug = $deepestReverbCategory->getData('reverb_category_slug');
 
         // We expect category slug to be populated, but check just in case
-        if (!empty($category_slug_array))
+        if (!empty($category_slug))
         {
-            $fieldsArray[self::CATEGORY_FIELD_NAME] = $category_slug_array;
+            $fieldsArray[self::CATEGORY_FIELD_NAME] = array($category_slug);
         }
         // If the only categories mapped are top-level Reverb categories, there will be no product type slug
         if (!empty($product_type_slug))
```

Comment:
this may be incorrect. product_type should _always_ be set. categories should _sometimes_ be set if there are mapped Reverb subcategories